lib/internal/bootstrap/realm.js

```diff
@@ -269,6 +269,12 @@ class BuiltinModule {
     }
   }
 
+  static setRealmAllowRequireByUsers(ids) {
+    canBeRequiredByUsersList = new SafeSet(ids.filter((id) => publicBuiltinIds.includes(id)));
+    canBeRequiredByUsersWithoutSchemeList =
+      new SafeSet(ids.filter((id) => !schemelessBlockList.has(id)));
```

Comment:
```suggestion
      new SafeSet(ArrayPrototypeFilter(ids, (id) => !schemelessBlockList.has(id)));
```

Reply:
I think this is actually OK because it's run before any user code, right? If so, we should add comment.

Reply:
Maybe, but it's inside a static function so it *could* be ran multiple times, and realms could potentially be created during runtime?

Reply:
It runs before the user codes. But yes, it is a static method that can run at a later time. I've applied the suggested change. Thanks for the suggestion!